saw-core/src/Verifier/SAW/Term/Pretty.hs

```diff
@@ -114,28 +114,19 @@ depthAllowed _ _ = True
 
 -- | Precedence levels, each of which corresponds to a parsing nonterminal
 data Prec
-  = PrecNone   -- ^ Nonterminal 'Term'
+  = PrecNone   -- ^ Nonterminal 'Term' or "sepBy(Term, ',')"
```

Comment:
Minor Haddock nit: text enclosed in double quotes denote links to names of modules, which I don't think you intended. Use this to make it render using a plain monospace font:

```suggestion
  = PrecNone   -- ^ Nonterminal 'Term' or @sepBy(Term, \',\')@
```

Escaping the single quotes is also necessary to prevent Haddock from trying to hyperlink `,` as an identifier name.